cpp/vcpkg.json

```diff
@@ -158,7 +158,7 @@
     { "name": "double-conversion", "version": "3.2.1" },
     { "name": "fizz", "version-string": "2022.10.31.00" },
     { "name": "fmt", "version": "9.1.0#1" },
-    { "name": "folly", "version-string": "2022.10.31.00#3" },
+    { "name": "folly", "version-string": "2023.05.22.00" },
```

Comment:
Could you move this to the top of this array to not confuse it with the comments on line 76.

Reply:
Done in 96ad7ddc10107f0a1c19b175bfaf43dcb8855c39. Let me know if this must be adapted.